sr_hand_health_report/src/sr_hand_health_report/backlash_check.py

```diff
@@ -0,0 +1,187 @@
+                    result['backlash_check'][joint.joint_name] = joint_result
+
+            self.switch_controller_mode("position")
+            self.move_finger_to_side(finger_object, 'left')
+
+        self._result = result
+        rospy.logerr(result)
+
+    def move_finger_to_side(self, finger_object, side):
+        angle = math.radians(-20) if side == 'right' else math.radians(20)
+        angle *= self._side_sign_map[finger_object.finger_name]
+        finger_object.joints_dict['J4'].move_joint(angle, 'position')
+
+    def wiggle_joint(self, joint):
+        test_times = []
+        total_time = None
+        test_count = 10  # number of iterations
+
+        test_start_time = rospy.get_rostime()
+        timeout_condition = True
+
+        while len(test_times) < test_count and timeout_condition and not rospy.is_shutdown():
+
+            success = True
+            test_time = rospy.get_rostime()
+            difference = 0
+            starting_position = joint.get_current_position()
+            start_time = rospy.get_rostime()
+            joint.move_joint(-self.FINGER_PWM, 'effort')
+            current_time = 0
+            while abs(difference) < self.THRESHOLD and not rospy.is_shutdown():
+                difference = joint.get_current_position() - starting_position
+                current_time = rospy.get_rostime() - start_time
+                time.sleep(0.01)
+                if current_time > rospy.Duration.from_sec(self.WIGGLING_TIME):
+                    test_time += rospy.Duration.from_sec(self.WIGGLING_TIME)
+                    success = False
+                    break
+
+            difference = 0
+            starting_position = joint.get_current_position()
+            start_time = rospy.get_rostime()
+            joint.move_joint(self.FINGER_PWM, 'effort')
+            current_time = 0
+            while abs(difference) < self.THRESHOLD and not rospy.is_shutdown():
+                difference = joint.get_current_position() - starting_position
+                current_time = rospy.get_rostime() - start_time
+                time.sleep(0.01)
+                if current_time > rospy.Duration.from_sec(self.WIGGLING_TIME):
+                    test_time += rospy.Duration.from_sec(self.WIGGLING_TIME)
+                    success = False
+                    break
+
+            timeout_condition = rospy.get_rostime() - test_start_time < rospy.Duration.from_sec(3 * self.WIGGLING_TIME)
+
+            if success:
+                test_time = rospy.get_rostime() - test_time
+                test_times.append(test_time)
+
+                if not total_time:
+                    total_time = test_time
+                else:
+                    total_time += test_time
+
+        joint.move_joint(0, 'effort')
+
+        result = {}
+        times = [time_entry.nsecs/10e9 for time_entry in test_times]
+        result['std'] = float(np.std(times))
```

Comment:
Why casting it to a float since it's the default returned type?

Reply:
Same for below

Reply:
if I try without casting then **yaml.representer.RepresenterError: ('cannot represent an object', 54.490965362364875)** is thrown. Is there no difference between python builtin float and numpy.float64?

Reply:
There is a difference, but I wrongly thought that yaml would handle that for you! So nevermind my comment, but please add a comment to explain why you recast it, that will be useful to the next person who needs to modify this file